js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -419,17 +422,35 @@ const AssetGraphExplorerWithData = ({
       shortcutFilter={(e) => e.altKey && e.code === 'KeyE'}
     >
       {expandedGroups.length === 0 ? (
-        <Button
-          title="Expand all groups"
-          icon={<Icon name="unfold_more" />}
-          onClick={() => setExpandedGroups(allGroups)}
-        />
+        <Tooltip
+          content={
+            <Box flex={{direction: 'row', gap: 4, alignItems: 'center'}}>
+              Expand all groups <KeyboardTag tooltip={true}>⌥E</KeyboardTag>
+            </Box>
+          }
+        >
+          <Button
+            title="Expand all groups"
+            icon={<Icon name="unfold_more" />}
+            onClick={() => setExpandedGroups(allGroups)}
+            style={{background: colorBackgroundDefault()}}
+          />
+        </Tooltip>
       ) : (
-        <Button
-          title="Collapse all groups"
-          icon={<Icon name="unfold_less" />}
-          onClick={() => setExpandedGroups([])}
-        />
+        <Tooltip
+          content={
+            <Box flex={{direction: 'row', gap: 4, alignItems: 'center'}}>
+              Collapse all groups <KeyboardTag tooltip={true}>⌥E</KeyboardTag>
```

Comment:
For boolean props with `true` values, you can just leave off the value:

```
<KeyboardTag tooltip>...</KeyboardTag>
```

Reply:
fixed now too